README.rst

```diff
@@ -48,9 +48,9 @@ To install the latest stable version of MNE-Python, you can use pip_ in a termin
 
     pip install -U mne
 
-**Note** that MNE-Python 0.17 was the last release to support Python 2.
-MNE-Python 0.18 only works under Python 3, and MNE-Python 0.19 requires
-Python 3.5 or higher.
+- MNE-Python 0.17 was the last release to support Python 2.7
+- MNE-Python 0.18 requires Python 3.5 or higher
```

Comment:
Why not mention 0.18-0.20? It's not important though.

Reply:
We're only mentioning the points at which requirements change